lib/screens/app/wallet/dashboard.dart

```diff
@@ -85,7 +85,7 @@ class _DashboardState extends State<Dashboard> {
             ),
             borderRadius: BorderRadius.circular(8),
           ),
-          padding: EdgeInsets.all(7),
+          padding: EdgeInsets.fromLTRB(7, 7, 7, 7),
```

Comment:
Question: How is this different? 
`EdgeInsets.all(7)` from `EdgeInsets.fromLTRB(7, 7, 7, 7)`

Reply:
Not different but a little easier to play around with

Reply:
Happy to change that back